### ISSUE TITLE
Does support the Kerberos based authentication?

### ISSUE THREAD

**Issue by nellaivijay** [linkedin/Burrow#424]
Does support the connectivity to kafka cluster if it kerberosized?

**Comment by cluyihunter** [comment]
yes, but the support is limited, at least based on my experience
related to issue #333 
try setting handshake-first=true under sasl 
and noverify=true for tls
no such support for zookeeper servers though

**Comment by jrevillard** [comment]
I'm also interested in keytab authentication... any update please ?

**Comment by rja1** [comment]
Didn't ever see an update here. So, can Burrow auth using SASL_PLAINTEXT + Keytab ?

Thanks much!

**Comment by jrevillard** [comment]
As far as I know, no... but sarama (which is used by Burrow does): since version 1.2.3 (Add kerberos support (https://github.com/Shopify/sarama/pull/1366))

It seems that the options needs to be exposed :1st_place_medal: 
https://github.com/Shopify/sarama/pull/1366/files#diff-b4bda758a2aef091432646c354b4dc59R532

Apparently, this has to be done here: https://github.com/linkedin/Burrow/blob/334be9125dcb3d0b5cddbee3f69cb956419d5a9e/core/internal/helpers/sarama.go#L118

Best,
Jerome
